Square.cs

```diff
@@ -0,0 +1,48 @@
+
+
+class Square : Rectangle
+{
+    int EdgeLength;
+    
+    public Square(bool FillShape, char Char, int EdgeLength) : base(FillShape, Char, EdgeLength, EdgeLength)
+    {
+        this.EdgeLength = EdgeLength;
+    }
+
+    public override void UpdateShape()
```

Comment:
The function should only update the shape rather than contain the whole process of updating.
This comment is relevant to other places in the code too.